Pipeline/RunIDZCodeReview/RunCodeReview.groovy

```diff
@@ -89,9 +100,15 @@ else
 	}
 	syslib = syslib.toUnique()
 
+	//If no SYSLIB found and no SYSLIB passed in codereview_syslib and no Property Group file passed, fails and exits
+	if (syslib.size() == 0 && !props.codereview_syslib && !props.codereview_PropertyGroupFile) {
```

Comment:
Rather then
```
syslib.size() == 0 && !props.codereview_syslib && !props.codereview_PropertyGroupFile
```
I think, it should be enough to either specify `codereview_syslib` or `codereview_PropertyGroupFile`
so. Or am I working on a wrong assumption?
```
syslib.size() == 0 && (!props.codereview_syslib || !props.codereview_PropertyGroupFile)
```

Reply:
Bare with me ... I looked at this the wrong way ...